src/wallet.ts

```diff
@@ -193,6 +193,30 @@ export function handleDeposit(event: Deposit): void {
     }
 }
 
+export function handleDeposited(event: Deposited): void {
+    let multisigAddr = event.params.recipient
+    let wallet = Wallet.load(multisigAddr.toHex())
+
+    if(wallet != null) {
+        wallet.balanceGaugeGSN = wallet.balanceGaugeGSN.plus(<BigInt> event.params.amount)
+        wallet.save()
+    } else {
+        log.warning("handleDeposited::Wallet {} not found", [multisigAddr.toHexString()])
```

Comment:
Warning is not necessary here. RelayHub events may not involve a Multis wallet all the time